guides/common/modules/proc_configuring-the-mcp-server-for-project.adoc

```diff
@@ -18,14 +19,21 @@ endif::[]
 ----
 $ podman pull {mcp-server-image}
 ----
-. Run the container, binding it to `127.0.0.1`:
+. Run the container:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 $ podman run --interactive --tty --publish 127.0.0.1:8080:8080 \
+  --volume _/Path/To/My/CA_Bundle.pem_:/app/ca.pem:ro \
```

Comment:
On OSes with selinux policies that might be not enough. At least our testing also required `Z` option for along with `ro`.
```suggestion
  --volume _/Path/To/My/CA_Bundle.pem_:/app/ca.pem:ro,Z \
```

Reply:
I added the option along with an explanation. Can you please review @ofedoren?